spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "User views all lists" do
+  scenario "successfully" do
+    create(:list, title: "Best Sci Fi")
+    book = create(:book, cover_image_url: "http://placehold.it/140x100")
+    list_2 = create(:list, title: "Based on a true story")
+    list_2.books << book
+
+    visit lists_path
+
+    expect(page).to have_content("Best Sci Fi")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.